File: CarHomeWidget/src/main/java/com/anod/car/home/ReenableCarModeReceiver.java
package com.anod.car.home;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;

public class ReenableCarModeReceiver extends BroadcastReceiver {

	@Override
	public void onReceive(Context context, Intent intent)
	{
		Log.d("CarHomeWidget", "onReceive car mode reenable: "+intent);
		// TODO
	}

}

File: app/src/main/java/com/anod/car/home/ReenableCarModeReceiver.kt
package com.anod.car.home

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log

class ReenableCarModeReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        Log.d("CarHomeWidget", "onReceive car mode re-enable: $intent")
    }

}